test/__init__.py

```diff
@@ -212,9 +216,11 @@ def __init__(self):
         self.client = None
         self.conn_lock = threading.Lock()
         self.is_data_lake = False
-
         if COMPRESSORS:
             self.default_client_options["compressors"] = COMPRESSORS
+        if MONGODB_API_VERSION:
```

Comment:
Instead, should we check for `REQUIRE_API_VERSION` and then instantiate the `server_api` object using `MONGODB_API_VERSION`?

Reply:
This is contingent on us keeping the `REQUIRE_API_VERSION` envvar. Seems like we might not need it?

Reply:
See my comment above about the difference between the two variables: https://github.com/mongodb/mongo-python-driver/pull/536/files/cfd91bd63ec0dcfb2becdeb2a6481861caf0af7e#r553177142